specificatie/genereervariant/openapi.json

```diff
@@ -314,6 +314,72 @@
               "type": "string"
             }
           },
+          {
+            "name": "postcode",
+            "in": "query",
+            "description": "Postcode behorende bij een nummeraanduiding.",
+            "required": false,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "pattern": "^[1-9]{1}[0-9]{3}[a-zA-Z]{2}$",
+              "type": "string",
+              "example": "6922KZ"
+            }
+          },
+          {
+            "name": "huisnummer",
+            "in": "query",
+            "description": "Huisnummer behorende bij een nummeraanduiding.",
```

Comment:
kunnen we dit veranderen in: het huisnummer van het adres

Reply:
zie mijn opmerking bij postcode